src/components/BlockedScenarios/BlockedScenarios.tsx

```diff
@@ -1,28 +1,55 @@
 import { ReactComponent as P2pUnavailable } from '@/assets/p2p-unavailable.svg';
+import {
+    DerivLightIcCashierBlockedIcon,
+    DerivLightIcCashierLockedIcon,
+    DerivLightIcCashierUnderMaintenanceIcon,
+} from '@deriv/quill-icons';
 import { Localize } from '@deriv-com/translations';
 import { ActionScreen, Button, Text, useDevice } from '@deriv-com/ui';
 import { URLConstants } from '@deriv-com/utils';
 
 type TBlockedScenariosObject = {
     [key: string]: {
-        actionButton: JSX.Element;
+        actionButton?: JSX.Element;
         description: JSX.Element;
         icon: JSX.Element;
         title: JSX.Element;
     };
 };
 
 const BlockedScenarios = ({ type }: { type: string }) => {
-    const { isDesktop } = useDevice();
+    const { isMobile } = useDevice();
 
-    const buttonTextSize = isDesktop ? 'sm' : 'md';
+    const buttonTextSize = isMobile ? 'md' : 'sm';
+    const iconSize = isMobile ? 96 : 128;
 
     // TODO: change redirection when account switcher is implemented
     const openDerivApp = () => {
         window.open(URLConstants.derivAppProduction, '_blank')?.focus();
     };
 
+    // TODO: implement this function to open live chat
+    const openLiveChat = () => {};
+
     const blockedScenarios: TBlockedScenariosObject = {
+        cashierLocked: {
```

Comment:
i asked yunus what we could show here, and he had shown me this:

![Screenshot 2024-07-10 at 2 56 37 PM](https://github.com/deriv-com/p2p/assets/103412909/3a2a19ff-7513-4770-ba7e-a726ea2265ef)